lib/movable_ink/aws/ec2.rb

```diff
@@ -255,6 +255,24 @@ def assign_ip_address(role:)
           })
         end
       end
+
+      def verify_elastic_ip_address(public_ip:)
+        expected_errors = [
+          MovableInk::AWS::Errors::ExpectedError.new(Aws::EC2::Errors::InvalidAddressNotFound, [/Address \'#{public_ip}\' not found./])
+        ]
+        begin
+          run_with_backoff(expected_errors: expected_errors) do
+            result = ec2.describe_addresses({
```

Comment:
`result` isn't used, you can just run `ec2.describe_addresses()` w/o assignment.

Reply:
good idea!